src/utils/config_filters.hpp

```diff
@@ -67,6 +67,6 @@ bool int_matches_if_present_or_negative(
 bool string_matches_if_present(
 	const config& filter, const config& cfg, const std::string& attribute, const std::string& def);
 
-bool bool_or_empty(const config& filter, const config& cfg, const std::string& attribute);
+bool matches_if_attribute_boolean_or_missing(const config& filter, const config& cfg, const std::string& attribute);
```

Comment:
Remove this part from the PR. It is not in scope.

If you must open a different PR for it but I'd rather you didn't.

Reply:
removed